src/chains.jl

```diff
@@ -346,11 +346,14 @@ function Base.show(io::IO, chains::Chains)
 end
 
 function Base.show(io::IO, mime::MIME"text/plain", chains::Chains)
-    print(io, "Chains ", chains, ":\n\n", header(chains))
+    println(io, "Chains ", chains, ":\n\n", header(chains))
```

Comment:
Is codecov right? Do we not test this function?

Reply:
Yes, it seems there was no test. I added one.